standard/clause_7_core.adoc

```diff
@@ -54,114 +57,147 @@ Resources exposed through an OGC API are accessed through a Universal Resource I
 |===
 ^|**Path Template** ^|**Resource**
 2+^|**Common**
-|{root}/ |Landing page
-|{root}/api |API Description
-|{root}/conformance |Conformance Classes
-|{root}/collections |Metadata describing the spatial collections available from this API.
-|{root}/collections/{collectionId} |Metadata describing the collection which has the unique identifier `{collectionId}`
+|<<landing-page,{root}/>> |Landing page
+|<<api-definition,{root}/api>> |API Description
```

Comment:
A concrete path is neither mandated in Common nor in Features isn't it? Suggest to harmonize.

Reply:
Clarified in telco on 20191218 to add (optional) to the /api path.